templates/seafile-notification-server

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>seafile-notification-server</Name>
+  <Repository>seafileltd/notification-server:12.0-latest</Repository>
+  <Registry>https://hub.docker.com/r/seafileltd/notification-server/</Registry>
+  <Network>seafile-net</Network>
+  <MyIP/>
+  <Shell>sh</Shell>
+  <Privileged>false</Privileged>
+  <Support>https://forums.unraid.net/topic/87798-support-selfhostersnets-template-repository/</Support>
+  <Project>https://github.com/NeoPlayer13/unRAID-CA-templates</Project>
+  <Overview>Notification Server for Seafile&#xD;
+&#xD;
+Converted By Community Applications   Always verify this template (and values)  against the support page for the container&#xD;
+&#xD;
+https://hub.docker.com/r/seafileltd/notification-server/&#xD;
+&#xD;
+////////////////////////////////////////////////////////////////////////////////////////&#xD;
+///Remenber to modify the file seafile.conf with the correct values of the notification server///&#xD;
+///////////////////////////////////////////////////////////////////////////////////////&#xD;
+https://manual.seafile.com/latest/deploy/notification-server&#xD;
+https://manual.seafile.com/latest/config/seafile-conf/#notification-server-configuration&#xD;
+</Overview>
+  <Category>Backup: Cloud:</Category>
+  <WebUI>http://[IP]:[PORT:8083]/ping</WebUI>
```

Comment:
If it's just a ping endpoint, that's not really a Web UI that the user would want to visit

```suggestion
```

Reply:
Yeah, is not a webui but is a faster way to verify if the container was setup correctly. The user avoids needing to type manually the URL.

Reply:
What would the URL be? Why not use that?